examples/docker.yml

```diff
@@ -41,7 +41,7 @@ services:
      - /var/lib/docker:/var/lib/docker
      - /lib/modules:/lib/modules
 files:
-  - path: etc/docker/daemon.json
+  - path: containers/services/docker/rootfs/etc/docker/daemon.json
```

Comment:
this is not the right way to do this. The container rootfs is off limits as it may change once we switch to the containerd storage. it's best to bind mount the json file into the docker container